Client/Frontend/Autofill/CreditCard/ViewComponents/CreditCardInputField.swift

```diff
@@ -11,17 +11,26 @@ enum CreditCardInputType {
 }
```

Comment:
A couple issues I noticed, and will put it here so a thread can continue.

- on concealing the CC card number, the font color doesn’t go back to what it was before
- when hitting copy on the name or date expiration fields, it doesn’t copy those values actually. so let’s say I hit copy on card number, then copy on expiration and paste, it’ll paste the card number instead